lib/components/user/monitored-trip/trip-status-rendering-strategies/active-trip-renderer.js

```diff
@@ -38,23 +38,39 @@ export default function activeTripRenderer ({
     if (tripStatus === REALTIME_STATUS.ON_TIME) {
       // about on time
       data.panelBsStyle = 'success'
-      data.headingText = 'Trip is in progress and is about on time.'
+      data.headingText = <FormattedMessage id='components.TripStatusRenderers.active.onTimeHeading' />
     } else if (tripStatus === REALTIME_STATUS.LATE) {
       // delayed
       data.panelBsStyle = 'warning'
-      data.headingText = `Trip is in progress and is delayed ${deviationHumanDuration}!`
+      data.headingText = <FormattedMessage
+        id='components.TripStatusRenderers.active.delayedHeading'
+        values={{
+          formattedDuration: <FormattedDuration
+            values={Math.abs(arrivalDeviationSeconds)}
+          />
+        }}
+      />
     } else {
       // early
       data.panelBsStyle = 'warning'
-      data.headingText = `Trip is in progress and is arriving ${deviationHumanDuration} earlier than expected!`
+      data.headingText = <FormattedMessage
+        id='components.TripStatusRenderers.active.earlyHeading'
+        values={{
+          formattedDuration: <FormattedDuration
+            values={Math.abs(arrivalDeviationSeconds)}
+          />
+        }}
+      />
     }
   } else {
     data.panelBsStyle = 'info'
-    data.headingText = 'Trip is in progress (no realtime updates available).'
+    data.headingText = <FormattedMessage id='components.TripStatusRenderers.active.noDataHeading' />
   }
 
-  data.bodyText =
-    `Trip is due to arrive at the destination at ${tripEndMoment.format(timeFormat)}.`
+  data.bodyText = <FormattedMessage
+    id='components.TripStatusRenderers.active.description'
+    values={{arrivalTime: tripEndMoment.format(timeFormat)}}
```

Comment:
Sorry I missed this one, remove the `moment` variable and add the time format to the language string instead.